apps/ex_shop/priv/repo/migrations/20160212073714_create_line_item.exs

```diff
@@ -3,7 +3,7 @@ defmodule ExShop.Repo.Migrations.CreateLineItem do
 
   def change do
     create table(:line_items) do
-      add :product_id, references(:not_products)
+      # add :product_id, references(:not_products)
```

Comment:
Lets have both variant and product id, variant_id will be used for all the work but product_id comes in handy in some cases :)